types/ffmpeg-static/ffmpeg-static-tests.ts

```diff
@@ -1,3 +1,5 @@
-import * as ffmpegStatic from 'ffmpeg-static';
+import * as pathToFfmpeg from 'ffmpeg-static';
 
-ffmpegStatic.path;
+if (typeof pathToFfmpeg !== "string") {
```

Comment:
The test code is not actually run, only checked against the compiler. This will not prove that `pathToFfmpeg` is a string.

Instead I think you want to use:
```js
pathToFfmpeg; // $ExpectType string
```